integration/unplugin/src/index.ts

```diff
@@ -1,4 +1,5 @@
 import { TransformResult, createUnplugin } from 'unplugin';
+// @ts-ignore
```

Comment:
This doesn't seem to be necessary. Right?

Reply:
Hm yeah it looks like it works without this as well. That's weird because earlier it was throwing a type error that `@danielx/civet` is not available to `integration/unplugin`